### PR TITLE
Update apps.yml

### PR DESCRIPTION
**Pull request by sabeehmurtaza9**

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -1101,3 +1101,14 @@
     - "database"
     - "design"
   icon: "collectie.png"
+-
+  name: "Calculator"
+  description: "A Simple Calculator with Google UI"
+  website: "https://softvilla.com.pk"
```
Comment (kevinsawicki) [inline _data/apps.yml]: When I try to visit the website I see the following warning:

screen shot 2016-05-28 at 10 17 36 am

File: _data/apps.yml [hunk 2]
```diff
@@ -1101,3 +1101,14 @@
...
+  repository: "https://github.com/sabeehmurtaza9/Calculator/"
```
Comment (kevinsawicki) [inline _data/apps.yml]: The code of your application does not appear to be in this repository, is it supposed to be there?

Comment (sabeehmurtaza9) [inline _data/apps.yml]: Sorry i forgot to upload the files there. Please check it again. My mistake.

**Comment by jlord** [comment]
Looks like https://github.com/electron/electron.atom.io/pull/314 replaces this one, closing out.

**Pull request closed without merging**